guides/v2.2/cloud/project/project-conf-files_services-mysql.md

```diff
@@ -108,3 +94,62 @@ relationships:
 
 {: .bs-callout-info }
 If you configure one MySQL user, you cannot use the [`DEFINER`](http://dev.mysql.com/doc/refman/5.6/en/show-grants.html) access control mechanism for stored procedures and views.
+
+## Connect to the database
+
+Accessing the MariaDB database directly requires you to use a SSH to log in to the remote server, and connect to the database with the following credentials:
+
+-  For Starter
+
+   ```bash
+   mysql -h database.internal -u <username>
+   ```
+
+-  For Pro, use the db, username, and password from the [service relationship]({{page.baseurl}}/cloud/project/project-conf-files_services.html#service-relationships):
+
+   ```bash
+   mysql -h<db> -p<number> -u<username> -p<password>
+   ```
+
+{:.procedure}
+Retrieve MySQL connection details and login credentials:
```

Comment:
This procedure doesn't seem necessary. It repeats how to print relationships, which is in other places in the cloud docs. Plus, it is a one step...because the SSH was stated before. Pro users need a little more information, but you provided a link above. To me, this seems like a lot of redundancy...maintenance.

Reply:
I added it here based on updated to the RabbitMQ. It's explicit information about how to connect to the database so user doesn't have to go anywhere else.  Although it might be redundant, it is for user ease of use. In both the Best practices for deployment, and the updated Go live checklist, there are guidelines to check the database. 

In terms of maintenance, I think need to go back and fix the service relationships section to provide the service relationship information that can be included rather than repeated. For now, I want to get this PR published.

Reply:
I do not disagree with needing to understand how to connect to the database. It is the way this procedure is set up, and you have a good point. The RabbitMQ procedure seems a little more helpful. This is a case where Pro differs, so working that in to the procedure would help.